.travis.yml

```diff
@@ -6,20 +6,20 @@ env: TRAVIS=1
 jobs:
     include:
         - name: "Tests"
-          if: type = push
+          if: type = pull_request
           services: mongodb
           install:
               - pip install cython
               - pip install . -r requirements-dev.txt
           script: py.test -rxs tests/
 
         - name: "Tests & Coverage"
-          if: NOT type = push
+          if: NOT type = pull_request
           services: mongodb
           install:
               - pip install cython
               - pip install -r requirements-dev.txt -e .
-          script: py.test --cov=scout -rxs tests/
+          script: py.test --cov=scout -rxs tests/commands/
```

Comment:
I think that as soon as you revert this and update the changelog we are good to merge!

Reply:
reverted